src/cli/cli.cc

```diff
@@ -1409,6 +1473,12 @@ int main (const int argc, const char* argv[]) {
       writeFile(paths.pathResourcesRelativeToUserBuild / "Credits.html", credits);
     }
 
+
+    // used in multiple if blocks, need to declare here
+    auto android_enable_standard_ndk_build = settings["android_enable_standard_ndk_build"] == "true";
```

Comment:
ideally, camel casing is preserved throughout the file for all identifiers and functions